source/documentation/administration_guide/topics/proc_migrating_she_host_to_different_cluster.adoc

```diff
@@ -0,0 +1,30 @@
+// module included:
+// doc-Administration_Guide/chap-Hosts.adoc
+
+[id='migrating_she_host_to_different_cluster_{context}']
+= Migrating a self-hosted engine host to a different cluster
+
+You cannot migrate a host that is configured as a self-hosted engine host to a data center or cluster other than the one in which the self-hosted engine virtual machine is running. All self-hosted engine hosts must be in the same data center and cluster.
+
+You need to disable the host from being a self-hosted engine host by undeploying the self-hosted engine.
```

Comment:
I'd change to `You need to disable the host from being a self-hosted engine host by undeploying the self-hosted engine configuration from the host.` Just to make sure that admins won't destroy the whole hosted engine ...

Reply:
Done.